app/components/works/show/attached_file_component.rb

```diff
@@ -24,10 +24,6 @@ def path
       def can_download?
         !attached_file.in_globus?
```

Comment:
It's not clear to me why we delegate some bits to `attached_file` but not others. Could remedy this by delegating `:file` and `:in_globus?` (for lines 21 and 25).

Reply:
Likely oversight.  Delegated both of those to `attached_filed` and updated methods as needed.